script.js

```diff
@@ -1,4 +1,246 @@
+  }
+  return myOutputValue;
+};
+
+// Korean Version
+// most recent winner is the ultimate winner when there is a draw
+var playKoreanMode = function (userName, input) {
+  currentGameMode = "Korean version";
+  myOutputValue = `Welcome, Player ${userName}! <br> You are now in KOREAN version of scissor paper stone. <br> Please enter 'scissors', 'paper', or 'stone' to play the game.`;
+  var computerChoose = randomItem();
+
+  // assigning emojis to the icon
+  var inputEmoji;
+  if (input == "scissors") {
+    inputEmoji = "✌";
+  } else if (input == "paper") {
+    inputEmoji = "✋";
+  } else if (input == "stone") {
+    inputEmoji = "✊";
+  }
+  var compEmoji;
+  if (computerChoose == "scissors") {
+    compEmoji = "✌";
+  } else if (computerChoose == "paper") {
+    compEmoji = "✋";
+  } else if (computerChoose == "stone") {
+    compEmoji = "✊";
+  }
+
+  // korean mode logic
+  // user win
+  if (
+    (input == "scissors" && computerChoose == "paper") ||
+    (input == "paper" && computerChoose == "stone") ||
+    (input == "stone" && computerChoose == "scissors")
+  ) {
+    recentWinner = "user";
+    myOutputValue = `Player ${userName}, you win! <br> You choose: ${input} ${inputEmoji}. <br> The computer chooses: ${computerChoose} ${compEmoji}.  <br> You need to play again and have draw to decide the ultimate winner.`;
+  } // if computer wins
+  else if (
+    (input == "scissors" && computerChoose == "stone") ||
+    (input == "paper" && computerChoose == "scissors") ||
+    (input == "stone" && computerChoose == "paper")
+  ) {
+    recentWinner = "computer";
+    myOutputValue = `Player ${userName}, you lose! <br> You choose: ${input} ${inputEmoji}. <br> The computer chooses: ${computerChoose} ${compEmoji}.  <br> You need to play again to decide the ultimate winner.`;
+  } // if it's draw
+  else if (input == computerChoose) {
+    if (recentWinner == "") {
+      recentWinner = "";
+      myOutputValue = `It is a DRAW. <br> You choose: ${input} ${inputEmoji}. <br> The computer chooses: ${computerChoose} ${compEmoji}. <br> Since no one wins the previous round, please play again.`;
+    } else if (recentWinner == "user") {
+      recentWinner = "";
+      return (myOutputValue = `It is a DRAW. <br> You choose: ${input} ${inputEmoji}. <br> The computer chooses: ${computerChoose} ${compEmoji}. <br> Since you are the recent winner, you WIN. <br> Congratulations Player ${userName}, you are the ultimate winner of muk-jji-ppa! `);
+    } else if (recentWinner == "computer") {
+      recentWinner = "";
+      return (myOutputValue = `It is a DRAW. <br> You choose: ${input} ${inputEmoji}. <br> The computer chooses: ${computerChoose} ${compEmoji}. <br> Since the computer is the recent winner, you LOST muk-jji-ppa! `);
```

Comment:
as most part of the korean game is the same as normal sps. you could use the normal sps function here and add on additional string to the turn for the muk-jji-ppa outcome
var mjpOuput= playNormalMode()+mjpOuput